semantics/src/main/scala/aqua/semantics/rules/ValuesAlgebra.scala

```diff
@@ -297,6 +297,10 @@ class ValuesAlgebra[S[_], Alg[_]: Monad](using
     valueToRaw(v).flatMap(
       _.flatTraverse {
         case ca: CallArrowRaw => ca.some.pure[Alg]
+        case ApplyPropertyRaw(value, IntoArrowRaw(name, arrowType, arguments)) =>
```

Comment:
(According to previous comment I don't think this should be done at all, but...)
Why put it here and not in `valueToRaw`? I believe `valueToRaw` should be the source of truth for representing values in `ValueRaw` tree. Putting this code here spreads responsability.
I wrote this method just to kinda "downcast" safely